ui/package.json

```diff
@@ -126,5 +129,10 @@
   "optionalDependencies": {
     "bufferutil": "^4.0.0",
     "utf-8-validate": "^5.0.0"
+  },
+  "peerDependencies": {
+    "@material-ui/core": "^1.0.0",
+    "mocha": "^5.0.0",
+    "karma": "^3.0.0"
```

Comment:
Why are we adding these peer dependencies?

Reply:
`mocha` and `karma` are a peer dependency for most of the `dependencies` we are using, This reduces the warnings on console by a significant amount.

Reply:
I'm not sure I understand. They are already specified in the list of dependencies so why list them in peerDependencies as well?

Reply:
Fixed it, Actually some of the depdendancies have been missing from `yarn.lock` peers.